lib/custom/zalgo.js

```diff
@@ -58,10 +58,10 @@ module['exports'] = function zalgo(text, options) {
   function heComes(text, options) {
     var result = '', counts, l;
     options = options || {};
-    options["up"] = options["up"] || true;
-    options["mid"] = options["mid"] || true;
-    options["down"] = options["down"] || true;
-    options["size"] = options["size"] || "maxi";
+    options["up"] =   typeof options["up"]   !== 'undefined' ? options["up"]   : true;
+    options["mid"] =  typeof options["mid"]  !== 'undefined' ? options["mid"]  : true;
+    options["down"] = typeof options["down"] !== 'undefined' ? options["down"] : true;
+    options["size"] = typeof options["size"] !== 'undefined' ? options["size"] : "maxi";
```

Comment:
Maybe I misunderstood, but, e.g. `options["up"] || true` part implied to me that `true` was meant to be the default value if `options["up"]` was not set.
But, if options["up"] is set to false, then:

    options["up"] || true
    == false || true
    == true

This was the problem I was having, I was unable to set things to false because the `|| true` was overriding them.